src/sentry/ingest/transaction_clusterer/rules.py

```diff
@@ -59,6 +59,7 @@ def write(self, project: Project, rules: RuleSet) -> None:
             p.delete(key)
             if len(rules) > 0:
                 p.hmset(key, rules)
+            p.execute()
```

Comment:
Without this call, the pipeline is never executed. I wrongly assumed that the context manager would call `execute` internally on exit.